.stylelintrc

```diff
@@ -6,6 +6,7 @@
   "rules": {
     "custom-property-pattern": "^(?!.+)",
     "scss/comment-no-empty": null,
+    "scss/operator-no-newline-after": null,
```

Comment:
This rule conflicts with Prettier behavior, so there is no point to keep it enabled